docs/apps/bioperl.md

```diff
@@ -31,15 +31,15 @@ BioPerl is licensed under the same terms as Perl itself which is dually-licensed
 
 ## Available
 
--   Puhti: Perl 5.30.0 with Bioperl 1.007002 
+-   Puhti: Perl 5.36.0 with Bioperl 1.7.8
```

Comment:
This does not seem to be the default version on Puhti. However, the `bioperl/1.007008` module seems to be identical to the `bioperl/1.7.8.` module (is the extra dot there on purpose in the modulefile?) @amsaren

Reply:
Some name confusion due to historical reasons. I think Kimmo M. has at some updated bioperl in module "bioperl/1.007008" but did not change the name. I created a new module with more informative name (points to same environment). I did not want to remove the other module as some users probably have it hard coded in their scripts. I now made made "bioperl/1.7.8" the default version, and also corrected the extra dot (thank for pointing this out).